TrackingTools/TrackFitters/plugins/KFFittingSmootherESProducer.cc

```diff
@@ -47,7 +47,8 @@ namespace {
       iRecord.get(pset_.getParameter<std::string>("Fitter"), fit);
       iRecord.get(pset_.getParameter<std::string>("Smoother"), smooth);
       
-     return std::shared_ptr<TrajectoryFitter>(new KFFittingSmoother(*fit.product(), *smooth.product(),pset_));
```

Comment:
@Dr15Jones what about this one. The pointer cast fails on this one.

Reply:
Try `std::make_unique<KFFittingSmoother>(*fit.product(), *smooth.product(),pset_);`

Reply:
OK